### ISSUE TITLE
Can not load the fp32 file directly！

### ISSUE THREAD

**Issue by meicale** [scott306lr/ResMLP_QAT#3]
Hi, It is great that you have made this repo happen and got so much done. I notice that the default fp32 model has a different dicts to huggingface and deit pretrained version. BUT, the repo doesn't contain the training scripts for the fp32 model. Do you mind sharing the pretrained model file. 
Thanks a lot!

**Comment by scott306lr** [comment]
Thank you for your interest in our repository. I regret to inform you that I am unable to share the modified pretrained model with you at this time. This repository is designed primarily to facilitate our own research, and it contains several modifications to the original model that are tailored to a unique data format required for our study.

I should note that if your primary concern is ResMLP QAT support with optimal accuracy, this repository may not serve your purposes to the fullest. We've observed a noticeable decline in accuracy due to some of the alterations we've implemented.

However, I'd like to inform you that I am planning to work on a new version of QAT support for MLP models this upcoming summer. One of the major features of this planned version is the ability to convert to TFLite.

Should you have any specific needs or suggestions, please feel free to share them with me. I am always open to new ideas and looking forward to possible ways to enhance this project.

**Comment by meicale** [comment]
Is it ok to get the pretrained model by using this repo? Or, would you like to try the [ original fp32 model ](https://huggingface.co/timm/resmlp_12_224.fb_in1k) and retrain it using this repo directly without modification. What is your suggestion if I want to do that?
I'd like to share my opinion about QResMLP, and as far as I know, the model after "ptq" with 8bit weight and 8bit activation has 74.93 accuracy in ImageNet classification.
